content/en/developing-pixelfed/intro.md

```diff
@@ -9,11 +9,12 @@ weight = 10
 +++
 
 ## Requirements
-- Git
-- PHP
-- Composer
-- Node
-- NPM
+- [Git](https://git-scm.com/downloads)
+- [PHP](https://www.php.net/manual/en/install.php)
+- [Composer](https://getcomposer.org/download/)
+- [Node](https://nodejs.org/en/download/)
+- [MySQL](https://www.mysql.com/downloads/) or [MariaDB](https://mariadb.org/download/?t=mariadb)
```

Comment:
PostgreSQL is technically not unsupported, although there have been historical issues with it due to less attention and testing. I'd mention Postgre here as an option